### ISSUE TITLE
TypeError: axios.get is not a function

### ISSUE THREAD

**Issue by amithgeorge** [foray1010/ignore-sync#598]
Installed v6. Even with a simple `.gitignore-sync` file, I get this error 

```
TypeError: axios.get is not a function
    at Object.getContentFile (<path-to-project>/node_modules/ignore-sync/src/utils/github.js:11:38)
    at <path-to-project>/node_modules/ignore-sync/src/generateIgnoreFile.js:25:21
    at Array.map (<anonymous>)
    at githubSourceFetcher (<path-to-project>/node_modules/ignore-sync/src/generateIgnoreFile.js:24:16)
    at <path-to-project>/node_modules/ramda/src/cond.js:64:30
    at <path-to-project>/node_modules/ramda/src/internal/_arity.js:11:19
    at _map (<path-to-project>/node_modules/ramda/src/internal/_map.js:7:19)
    at map (<path-to-project>/node_modules/ramda/src/map.js:83:14)
    at <path-to-project>/node_modules/ramda/src/internal/_dispatchable.js:50:15
    at Object.f2 [as map] (<path-to-project>/node_modules/ramda/src/internal/_curry2.js:34:14)
```

File 
```
[local]
client/dist

[github/gitignore]
Node.gitignore
Global/macOS.gitignore
```

**Comment by amithgeorge** [comment]
Seems the fix might be to require axios correctly - https://github.com/axios/axios#note-commonjs-usage


```
const axios = require('axios').default;
```

in https://github.com/foray1010/ignore-sync/blob/1ca3b5589445d8e92c38e463b598e2ecaa59d856/src/utils/github.js#L3

**Comment by foray1010** [comment]
They just released [v1.1.0](https://github.com/axios/axios/releases/tag/v1.1.0) with this [breaking change](https://github.com/axios/axios/pull/5030) an hour ago... That's why everything works fine for me but not you. I have just released `v6.0.1` to fix that.
